ingestion/portland_ingest.py

```diff
@@ -1,7 +1,7 @@
 """
```

Comment:
As a general comment for the file, nothing that will block merging but if we could add type hints for the functions that would be great to increase readability.